src/layouts/partials/DrawerMainMenu/MenuTokensList/MenuTokensList.jsx

```diff
@@ -156,8 +159,7 @@ export default class MenuTokensList extends PureComponent {
               </div>
               <div
                 styleName='addressTitle'
-                onTouchTap={((blockchain) => () =>  {
-                  this.handleScrollToBlockchain(blockchain)}
+                onTouchTap={((blockchain) => () => this.handleScrollToBlockchain(blockchain)
```

Comment:
[MINOR] Common recommendation of any linter: do not use fat-arrow methods inside JSX.